### DISCUSSION TITLE
makeAUC

### DISCUSSION THREAD

**Discussion by anne8rete** [LAPKB/Pmetrics#81] [Q&A]
Hi, 

We have an issue with makeAUC function - getting different values with different computers. 

In one AUC24h is 66.02 (Pmetrics 1.9.7, R version 4.0.2); other 88.3 (Pmetrics 2.0, however legacy code used, R version 4.1.1). 

Is there a way to know which AUC to trust? 

Thanks, 
Anne

**Comment by mnneely** [comment]
Can you check if you have 66.02 as a value anywhere else in the AUCs from Version 2? This may be related to a bug in sort order.

On Jul 13, 2022, at 8:25 AM, anne8rete ***@***.***> wrote:

﻿

Hi,

We have an issue with makeAUC function - getting different values with different computers.

In one AUC24h is 66.02 (Pmetrics 1.9.7, R version 4.0.2); other 88.3 (Pmetrics 2.0, however legacy code used, R version 4.1.1).

Is there a way to know which AUC to trust?

Thanks,
Anne

**Reply by anne8rete** [reply]
So we get the 88 with version 2, but also with version 1.9.7, just different computers running slightly different R versions.

What do you mean by sort order?

**Reply by anne8rete** [reply]
I've just asked from a colleague, manually calculating with using the posteriors the AUC is around 55.
